packages/e2e/cypress/integration/common/fileuploader.ts

```diff
@@ -39,15 +39,15 @@ When(
 );
 
 When('I drag and drop a file with file name {string}', (fileName: string) => {
-  cy.get('input[type=file]').selectFile(
-    {
-      contents: Cypress.Buffer.from('file contents'),
-      fileName,
-      lastModified: Date.now(),
+  cy.get('input[type=file]').trigger('drop', {
```

Comment:
Curious about this change. Looks like only one storage manager test is failing but this change affects all tests, could you explain a bit more about it? In [CI](https://github.com/aws-amplify/amplify-ui/actions/runs/6674643826/job/18141881100) failed test screenshot reveals failure due to the fact that 2 files were uploaded somehow, when the component is configured to only accept one. Maybe some cleanup from a previous test missing.

Reply:
That's a good point, can you take a look @ioanabrooks?

Reply:
Right but we still want to test both, can we find a way to cleanup between tests or split tests?

Reply:
Splitting makes sense to me, imo would improve our usage scenario coverage

Reply:
Do you mean add an additional test for successfuly uploading to public access but with select instead of drag/drop? I'm confused because there was only one test using this code and the scenario was "I should be able to click and drag any files" so I thought it was only testing drag/drop

Reply:
Actually it looks like this assertion is only used in one file, so I think the change makes sense.
![CleanShot 2023-10-31 at 09 34 25](https://github.com/aws-amplify/amplify-ui/assets/6165315/fa3000ce-bccd-4f83-974f-e6f2f6a3ae77)